app/services/subject_mapper.rb

```diff
@@ -108,18 +87,7 @@ class SubjectMapper
     },
   }.freeze
 
-  def self.is_further_education(subjects)
-    subjects = subjects.map { |subject| (subject.strip! || subject).downcase }
-    (subjects & SUBJECT_LEVEL[:ucas_further_education]).any?
-  end
-
-  def self.map_to_subject_name(ucas_subject)
-    res = (@ucas_rename[ucas_subject] || ucas_subject).capitalize
-
-    (res.sub "english", "English" || res)
-  end
-
-  class GroupedSubjectMapping
+  class StaticMapping
```

Comment:
Can we keep it at one class per file? I also feel like `SubjectMapper::StaticMapping` doesn't tell us much. Can we use naming that makes it clear what we are mapping?